### ISSUE TITLE
Wrong symbol for some three Articulata crinoid orders

### ISSUE THREAD

**Issue by pnovack-gottshall** [paleobiodb/navigator#108]
The "critter" symbols for several Infraorder Articulata crinoids (Orders Roveacrinida, Ampelocrinida, and Cyrtocrinida) are plotting as brachiopods instead of crinoids. Can you fix? (These orders are correctly "opinioned" as crinoids in the taxonomic opinions, and the orders Comatulida, Encrinida, Isocrinida, Millericrinida, and Uintacrinida are plotting correctly as crinoids.)

**Comment by markuhen** [comment]
This is clearly a disambiguation issue. All of these clades belong to the Articulata, which is also a clade name in Brachiopoda. I'm not sure how we handle other disambiguation problems in Navigator, but this is clearly the problem.

**Comment by vjpsyverson** [comment]
I think this issue has to be handled on the backend; as far as I can tell, Navigator is responding correctly and one of the tables called by occs/prevalence is messed up. I'm going to pass this on to @jpjenk to figure out why occs/prevalence is returning the wrong value for "img".

The mapping of images to taxa is handed up straight from the API. navigator.js makes [a call to occs/prevalence](https://paleobiodb.org/data1.2/occs/prevalence.json?limit=50&lngmin=-180.0&lngmax=180.0&latmin=-90.0&latmax=90.0&base_id=129759), which returns the key "img" with a numeric value. The crinoid orders with the problem return "img: 34", which if you pass it to taxa/thumb returns a picture of an articulate brachiopod. I suspect there are just a couple of bad values in the table.  (Oddly, most of the articulate crinoid orders seem to have [the correct image number](https://paleobiodb.org/data1.2/occs/prevalence.json?limit=50&lngmin=-180.0&lngmax=180.0&latmin=-90.0&latmax=90.0&interval_id=2&base_id=31590), 1752; I don't know what happened to those three!)

**Comment by mmcclenn** [comment]
Yes, this is definitely a name matching issue. I am looking into it now.
